### ISSUE TITLE
How can I run rocker noninteractively without the container automatically stopping?

### ISSUE THREAD

**Issue by themightyoarfish** [osrf/rocker#90]
https://github.com/osrf/rocker/blob/b1fae79fe83cdfba025450e304bdcef0064107fb/src/rocker/core.py#L247

This line here seems to pass `-it` to docker if `--mode non-interactive` is passed to rocker. It seems to me that actually makes the container start  in interactive mode. Isn't the `-d` flag what we want for detached execution?

**Comment by tfoote** [comment]
Non-interactive means that you don't get keyboard interactions. Detached is different. It could be extended to support detaching too.

**Comment by taihup** [comment]
Hi guys, pretty cool rocker!

+1 to "detached" feature :muscle:
